nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-canvas-utils.js

```diff
@@ -412,9 +412,9 @@
 
                 // create object whose keys are the parameter name and the values are the parameter values
                 var paramsObject = {};
-                params.forEach(function (v, k) {
-                    paramsObject[k] = v;
-                });
+                for(var pair of params.entries()) {
```

Comment:
Changing the "of" to "in" is not correct... I tried it and verified that using the "in" breaks deep linking. If you look here (in the Browser Compatibility section): https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams we see that "for...of" is the supported way to iterate over the GET params...